client-v2/src/components/FrontsEdit/Front.tsx

```diff
@@ -37,6 +37,7 @@ import { initialiseCollectionsForFront } from 'actions/Collections';
 
 const FrontContainer = styled('div')`
   display: flex;
+  min-height: 0;
```

Comment:
this is odd, my css skills aren't strong enough to fully understand why we'd do this...

Reply:
Apparently this is a firefox thing which is needed for the scroll bars to appear in a flex container. I'll add a comment to make this clearer.